grid2op/utils/l2rpn_idf_2023_scores.py

```diff
@@ -103,32 +104,41 @@ def __init__(
             nb_process_stats=nb_process_stats,
             scores_func={
                 "grid_operational_cost": L2RPNSandBoxScore,
+                "assistant_confidence": _AlertTrustScore,
+                # "assistant_cost": _AlertCostScore,
                 "new_renewable_sources_usage": _NewRenewableSourcesUsageScore,
             },
             score_names=["grid_operational_cost_scores",
+                         "assistant_confidence_scores",
+                         # "assistant_cost_scores",
                          "new_renewable_sources_usage_scores"],
             add_nb_highres_sim=add_nb_highres_sim,
         )
-        
-        assert(weight_op_score + weight_assistant_score + weight_nres_score==1.)
-        assert(all([weight_confidence_assistant_score>=0., weight_confidence_assistant_score<=1.]))
-        
+
+        test_weights = weight_op_score + weight_assistant_score + weight_nres_score
+        if test_weights != 1.0:
+            raise Grid2OpException(
+                'The weights of each component of the score shall sum to 1'
+            )
+        # assert(all([weight_confidence_assistant_score>=0., weight_confidence_assistant_score<=1.]))
```

Comment:
why is this assert removed ?

Reply:
I did not chage it, @laure-crochepierre  @GoubetClem any idea ?

I would say probably because we only use one sub-score for the Assistant Score and that this weight is not necessary anymore

Reply:
yes, I think you can undo it. It looks like a typo. It was just modified in the last commit [synchro merge with branch 1.9.2](https://github.com/rte-france/Grid2Op/pull/492/commits/984c01ee6ba413ed6296b6b9c5951d1894ae9258)

Reply:
The weight_confidence_assistant_score is not used anymore (since the AlertCostScore is removed) so I delete it as argument and deleted this assert as well